### PR TITLE
Split settings groups in `RftPlotter`

### PR DESCRIPTION
**Pull request by lindjoha**
Splits the previous `Selection` settings group into `PlotType` and `Ensembles`. 

---

### Contributor checklist

- [ ] :tada: This PR closes #ISSUE_NUMBER.
- [ ] :scroll: I have broken down my PR into the following tasks:
   - [ ] Task 1
   - [ ] Task 2
- [ ] :robot: I have added tests, or extended existing tests, to cover any new features or bugs fixed in this PR.
- [ ] :book: I have considered adding a new entry in `CHANGELOG.md`, and added it if should be communicated there.

### CODE REVIEW

File: webviz_subsurface/plugins/_rft_plotter/_views/_sim_vs_obs_view/_settings/_ensembles.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    def __init__(self, ensembles: List[str]) -> None:
+        super().__init__("Ensembles")
+        self._ensembles = ensembles
+
+    def layout(self) -> List[Component]:
+        return [
+            wcc.Dropdown(
+                id=self.register_component_unique_id(self.Ids.ENSEMBLES),
+                options=[{"label": ens, "value": ens} for ens in self._ensembles],
+                value=[self._ensembles[0] if len(self._ensembles) > 0 else None],
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_rft_plotter/_views/_sim_vs_obs_view/_settings/_ensembles.py:22]:
```suggestion
                value=[self._ensembles[0] if self._ensembles else None],
```
...if I'm not mistaken this should be possible and preferred (I'm actually a bit surprised pylint didn't complain - so maybe I'm wrong)

**Review by anders-kiaer — APPROVED** [review]
LGTM! One small comment only 🙂

File: webviz_subsurface/plugins/_rft_plotter/_views/_sim_vs_obs_view/_settings/_ensembles.py (hunk 1 above)
Comment (lindjoha) [reply webviz_subsurface/plugins/_rft_plotter/_views/_sim_vs_obs_view/_settings/_ensembles.py:22]: Good point!

Comment (lindjoha) [reply webviz_subsurface/plugins/_rft_plotter/_views/_sim_vs_obs_view/_settings/_ensembles.py]: I have fixed that many other places in the plugin as well.

**Pull request merged**